gapic-generator-cloud/templates/cloud/gem/_slashed.erb

```diff
@@ -26,9 +26,7 @@ module <%= gem.namespace.split("::")[-1] %>
 
     config_attr :credentials,  nil do |value|
       allowed = [::String, ::Hash, ::Proc, ::Google::Auth::Credentials, ::Signet::OAuth2::Client, nil]
-      if defined? ::GRPC
-        allowed += [::GRPC::Core::Channel, ::GRPC::Core::ChannelCredentials]
-      end
+      allowed += [::GRPC::Core::Channel, ::GRPC::Core::ChannelCredentials] if defined? ::GRPC
```

Comment:
I thought we were dispensing with IfUnlessModifier

Reply:
No, this was mentioned as a rule to exclude in an earlier message in error.

Reply:
oh, okay.